### PR TITLE
Fix/left aligned chat input

### PR DESCRIPTION
**Pull request by yurii-lubynets**
closes: https://github.com/sparkletown/internal-sparkle-issues/issues/809

### CODE REVIEW

**Comment by yurii-lubynets** [comment]
Now the Chat input looks good

Placeholder:
Screenshot 2021-07-16 at 12 22 04


Input text:
Screenshot 2021-07-16 at 12 22 20

File: src/components/molecules/ChatMessageBox/ChatMessageBox.scss [hunk 1]
```diff
@@ -13,6 +13,11 @@
 
   &__input {
     width: 100%;
+
+    .InputField__input {
```
Comment (0xdevalias) [inline src/components/molecules/ChatMessageBox/ChatMessageBox.scss:17]: Why is this class being added here? Presumably to override some centreing happening in this file or similar? This feels like there is possibly a better solution

Comment (yurii-lubynets) [reply src/components/molecules/ChatMessageBox/ChatMessageBox.scss:17]: @0xdevalias 

Screenshot 2021-07-16 at 16 03 40

I'm overriding the globals styles, adding the relevant indent, and fixing directly for this Chat input. WDYT?

**Comment by yurii-lubynets** [comment]
@sparkletown/engineering feel free for review

**Review by mike-lvov — APPROVED** [review]
Approved after my changes. @yurii-lubynets please, check the changes I made

**Comment by yurii-lubynets** [comment]
> Approved after my changes. @yurii-lubynets please, check the changes I made

got it! Thank you for help

**Pull request merged**